File: windows-samples/poslink/EposPosLinkExample/EposPosLinkExample/Models/Product.cs
using System.Collections.Generic;

namespace EposPosLinkExample.Models;

public record Product(int Id, string Name, decimal Price)
{
    public static List<Product> GetProducts() =>
    [
        new(1, "Apple", 0.99m),
        new(2, "Banana", 0.59m),
        new(3, "Orange", 0.79m),
        new(4, "Grapes", 2.49m),
        new(5, "Mango", 1.99m),
        new(6, "Peach", 1.29m),
        new(7, "Lemon", 1.29m),
        new(8, "Lime", 1.29m),
        new(9, "Strawberry", 1.29m),
        new(10, "Watermelon", 1.29m),
    ];
}

File: java-samples/EposPosLinkExample/src/main/java/com/example/eposposlinkexample/models/Product.java
package com.example.eposposlinkexample.models;

import java.math.BigDecimal;
import java.util.List;

public record Product(int id, String name, BigDecimal price, String emoji) {

    public static List<Product> getProducts() {
        return List.of(
                new Product(1, "Apple", new BigDecimal("0.99"), "🍎"),
                new Product(2, "Banana", new BigDecimal("0.59"), "🍌"),
                new Product(3, "Orange", new BigDecimal("0.79"), "🍊"),
                new Product(4, "Grapes", new BigDecimal("2.49"), "🍇"),
                new Product(5, "Mango", new BigDecimal("1.99"), "🥭"),
                new Product(6, "Peach", new BigDecimal("1.29"), "🍑"),
                new Product(7, "Lemon", new BigDecimal("1.29"), "🍋"),
                new Product(8, "Lime", new BigDecimal("1.29"), "🟢"),
                new Product(9, "Strawberry", new BigDecimal("1.29"), "🍓"),
                new Product(10, "Watermelon", new BigDecimal("1.29"), "🍉")
        );
    }
}
